packages/hooks/src/useServiceToken.ts

```diff
@@ -0,0 +1,24 @@
+import { useQuery } from '@deriv/api';
+import { TSocketRequestPayload } from '@deriv/api/types';
+
+type TServiceTokenPayload = TSocketRequestPayload<'service_token'>['payload'];
+
+/**
+ * Hook to get the service token for using 3rd party SDK
+ * @name useServiceToken
+ * @param payload to be sent while making the API call
+ * @returns response and its status
+ */
+const useServiceToken = (payload: TServiceTokenPayload) => {
+    const { data, ...rest } = useQuery('service_token', {
+        payload,
+        options: { retry: 3, enabled: Boolean(payload) },
```

Comment:
Hey man, since the payload is not fixed. Lets pass it from the component. Can checkout useVerifyEmail from api package.